css/courses.css

```diff
@@ -0,0 +1,97 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800&display=swap");
+body{
+    background-color: white;
+}
+
+
+/* Styling for the header */
+header{
+    margin-top:150px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 45px;
+    line-height: 61px;
+    color: #000000;
+    margin-bottom: 50px;
+}
+
+/* Styling for the card components */
+.cards-box{
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-auto-rows: auto;
+    grid-gap: 1.5rem;
+}
+.card-box{
+    overflow: auto;
+    margin-bottom: 100px;
+    width: 100%;
+}
+
+.card-image-container img{
+    display: block;
+    min-width: 200px;
+    min-height: 200px;
+    background-color: rgba(196, 196, 196, 0.65);
+    border-radius: 10px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5673604)